packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -992,7 +1027,36 @@ export class ContainerRuntime
 		return this._getAttachState();
 	}
 
-	public idCompressor: (IIdCompressor & IIdCompressorCore) | undefined;
+	private _idCompressor: (IIdCompressor & IIdCompressorCore) | undefined;
+	private pendingIdCompressorOps: IdCreationRange[] = [];
```

Comment:
Can you please add comments for `pendingIdCompressorOps` and `skipSavedCompressorOps`. It's not obvious what purpose they have.